adaptor/index.html

```diff
@@ -214,25 +212,25 @@
 </b>
 &nbsp;&nbsp;&nbsp;&nbsp;
 &nbsp;&nbsp;&nbsp;&nbsp;
-<a href="http://static.googleusercontent.com/media/www.google.com/en/us/support/enterprise/static/gsa/docs/admin/connectors/40/413/DeployingtheConnectorforFileSystems.pdf">Deploying Connector for File Systems 4.1.3</a><br>
+<a href="http://static.googleusercontent.com/media/www.google.com/en//support/enterprise/static/gsa/docs/admin/connectors/40/413/DeployingtheConnectorforDocumentum.pdf">Deploying Connector for Documentum 4.1.3</a><br>
```

Comment:
I was looking at why some of these have /en//support and others have /en/us/support, and I see that the docs page has different links entirely, and there are redirects to these links when the PDF is displayed. I think we should use the original links from the Connector documentation landing page, e.g.,

http://www.google.com/support/enterprise/static/gsa/docs/admin/connectors/40/413/DeployingtheConnectorforDocumentum.pdf

Since we're changing almost every line here, now is a good time to switch all the links.